fdbserver/DataDistribution.actor.cpp

```diff
@@ -310,6 +310,8 @@ struct DataDistributor : NonCopyable, ReferenceCounted<DataDistributor> {
 	// Optional components that can be set after ::init(). They're optional when test, but required for DD being
 	// fully-functional.
 	DDTeamCollection* teamCollection;
+	Reference<ShardsAffectedByTeamFailure> shardsAffectedByTeamFailure;
+	PromiseStream<RelocateShard> relocationProducer, relocationConsumer; // comsumer is a yield stream from producer
```

Comment:
The comment is not clear to me. Would you mind rewording this please?

Reply:
I'll try to do that. Basically, `relocationProducer` is the old `output` and `relocationConsumer` is the old `input`.